cloud-sql-proxy/README.MD

```diff
@@ -0,0 +1,49 @@
+# Cloud SQL I/O and Hive Metastore
+
+This initialization action installs a [Google Cloud SQL proxy](https://cloud.google.com/sql/docs/sql-proxy) on every node in a [Google Cloud Dataproc](https://cloud.google.com/dataproc) cluster. It also configures the cluster to store [Apache Hive](https://hive.apache.org) metadata on a given Cloud SQL instance.
+
+## Using this initialization action
+You can use this initialization action to create a Dataproc cluster using a shared hive metastore.:
+
+1. Using the `gcloud` command to create a new 2nd generation Cloud SQL intance (or use a previously created instance)
+
+    ```bash
+    gcloud sql instances create <INSTANCE_NAME> \
+        --tier db-n1-standard1 \
+        --activation-policy=ALWAYS
+    ```
+  a. Optionally create (or already have) other 2nd generation instances, which you wish to be accessible.
+2. Uploading a copy of this initialization action (`cloud-sql-proxy.sh`) to [Google Cloud Storage](https://cloud.google.com/storage).
```

Comment:
I would be happy to take this on a separate issue as part of larger cleanup, but I would prefer consistency with other components at this point.